ci/tests/general/general.py

```diff
@@ -262,6 +262,19 @@ def get_mountpoints(client):
         return mountpoints
 
     @staticmethod
+    def unmount_partition(root_client, partition):
+        """
+        Unmount a partition
+        :param root_client: ssh-connection
+        :param partition: Partition Dal object
+        :return: None
+        """
+        client_mountpoints = General.get_mountpoints(root_client)
+        for mountpoint in client_mountpoints:
+            if partition.mountpoint in mountpoint:
```

Comment:
When unmounting a partition you expect to only umount 1 mountpoint
With this implementation you could umount multiple mountpoints

I'd go for an exact match instead of 'in'